lib/rrule/rule.rb

```diff
@@ -87,6 +87,10 @@ def humanize
       Humanizer.new(self, options).to_s
     end
 
+    def has_end_limit?
```

Comment:
Nice, thanks! One nit - what about calling this method `is_finite?`? "limit" puts me in mind of a SQL numerical LIMIT and I think I might wonder if this method conveyed the same (i.e. if it meant "this RRULE has a COUNT").

Reply:
Yeah, good point. Updating..